<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAccount</class>
 <widget class="QDialog" name="DlgAccount">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/16x16_ico_png/ico_accounts.png</normaloff>:/resource/16x16_ico_png/ico_accounts.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnTest">
       <property name="text">
        <string>Test ...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccept">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>Servers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="3" column="0">
        <widget class="QGroupBox" name="grpSecure">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Secure server (TCP/SSL)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Host</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="edtHostSecure">
              <property name="toolTip">
               <string>Name of the news server computer or its IP address.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="edtPortSecure">
              <property name="toolTip">
               <string>News server port for secure (SSL) connection.</string>
              </property>
              <property name="text">
               <string>443</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="grpGeneral">
         <property name="title">
          <string>General server (TCP)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Host</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="edtHost">
              <property name="toolTip">
               <string>Name of the news server computer or its IP address.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="edtPort">
              <property name="toolTip">
               <string>News server port. </string>
              </property>
              <property name="text">
               <string>119</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="grpLogin">
         <property name="title">
          <string>Service requires login</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="edtUsername">
              <property name="toolTip">
               <string>If the server requires no authentication username can be empty otherwise this is the username given by the service provider.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="edtPassword">
              <property name="toolTip">
               <string>If the server requires no authentication username can be empty otherwise this is the password given by the service provider.</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabData">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Data files</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="edtDataPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnBrowse">
            <property name="text">
             <string>Browse ...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="chkCompression">
            <property name="text">
             <string>Use compressed headers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkPipelining">
            <property name="text">
             <string>Use command pipelining</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLineEdit" name="edtName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Connections</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QSpinBox" name="maxConnections">
        <property name="toolTip">
         <string>Maximum connections</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>maxConnections</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>grpGeneral</tabstop>
  <tabstop>edtHost</tabstop>
  <tabstop>edtPort</tabstop>
  <tabstop>grpSecure</tabstop>
  <tabstop>edtHostSecure</tabstop>
  <tabstop>edtPortSecure</tabstop>
  <tabstop>grpLogin</tabstop>
  <tabstop>edtUsername</tabstop>
  <tabstop>edtPassword</tabstop>
  <tabstop>chkCompression</tabstop>
  <tabstop>chkPipelining</tabstop>
  <tabstop>edtDataPath</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>btnTest</tabstop>
  <tabstop>btnAccept</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
